### PR TITLE
Optimizing compose files for Avalon CI

### PR DESCRIPTION
**Pull request by pankajgoyal2**
Signed-off-by: pankajgoyal2 <pankaj.goyal@intel.com>

### CODE REVIEW

File: bin/run_docker_test [hunk 1]
```diff
@@ -53,7 +53,8 @@ def main():
 
     # Search for compose file passed
     compose_file = _get_compose_file(args.compose_file)
-    test_service = _get_test_service(compose_file)
+    #test_service = _get_test_service(compose_file)
```
Comment (rranjan3) [inline bin/run_docker_test:56]: Is this a temporary change? If not remove it.

File: ci/docker-compose-fabric-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Intel Corporation
```
Comment (manju956) [inline ci/docker-compose-fabric-tests.yaml:1]: Update copyright to 2020

File: bin/run_tests [hunk 3]
```diff
@@ -33,13 +33,19 @@ FABRIC_PROXY_MODEL_SGX_TESTS=0
 main() {
 
     if [ "$DIRECT_MODEL_TESTS" == "1" ]; then
-        run_docker_test ./ci/avalon-direct-model-tests.yaml
+        run_docker_test ./ci/docker-compose-direct-model.yaml
     elif [ "$FABRIC_PROXY_MODEL_TESTS" == "1" ]; then
-        run_docker_test ./ci/avalon-fabric-proxy-model-tests.yaml
+	cd ci
```
Comment (dcmiddle) [inline bin/run_tests:38]: tab alignment

File: ci/docker-compose-direct-model-sgx.yaml [hunk 4]
```diff
@@ -15,21 +15,23 @@
...
+      # ZMQ socket port.
+      - 5555
+    volumes:
+      # Map aesm service on host to docker, required only in HW mode.
+      - /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket
+    devices:
+      - "/dev/isgx:/dev/isgx"
+
+  avalon-listener:
+    expose:
```
Comment (dcmiddle) [inline ci/docker-compose-direct-model-sgx.yaml:35]: tab alignment

**Pull request merged**